### ISSUE TITLE
1.5.1 Server files do not work both updating from 1.5 and from scratch

### ISSUE THREAD

**Issue by paynejj** [tracer4b/nomi-ceu#255]
**Pack Version: 1.5.1**

**Launcher:** server side, but I use ATLauncher

**Description:** 
TLDR: I recently started a nomifactory server on 1.5. After updating to 1.5.1, the server spams script errors to chat when any player joins. 

1. **First Boot:** FML pauses the boot to ask for confirmation on removing certain missing items from the world, I hit /fml continue to continue.  The server boots, and the error logged in chat is "No such member" or "Invalid Type" from line 8 of scripts/normalmode.zs, where it tries to import mods.gregtech.IControllerTile. Inventory sorting and other functions from CraftTweaker do not work.
2. **Subsequent Boots:** Server starts without doing anything abnormal, but when a player joins, chat is filled with errors from multiblocks_normal.zs, coming from multiple lines shown in screenshot. 

**Expected Behavior:** CraftTweaker does not crash, errors are not logged in chat

**Steps To Reproduce:**
1. Download and extract server files from curseforge or github
2. Copy world folder from v1.5 to v1.5.1 server files. 
3. Start server and log in

**Screenshots:**
The error shown in subsequent boots with literally everything I've done on the server in the background

![2023-02-17_11 55 37](https://user-images.githubusercontent.com/70377045/219746486-88b24542-5aa8-4e2e-a153-01d5bdc5c8ce.png)

**Comment by paynejj** [comment]
Update: The 1.5.1 server files do not work at all. Starting from scratch, no changes made by me,  the server crashes with this error: 

Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from CraftTweaker2 (crafttweaker)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/texture/TextureMap

**Comment by TheBlackSlayer** [comment]
Have the same problem with the wall of red text updating from 1.5 to 1.5.1, still working for me for now, no apparent error on the server log
[latest.log](https://github.com/tracer4b/nomi-ceu/files/10770385/latest.log)

**Comment by IntegerLimit** [comment]
Can you send the craftweaker.log, located in your minecraft folder?
Thanks.

**Comment by TheBlackSlayer** [comment]
Sure
[crafttweaker.log](https://github.com/tracer4b/nomi-ceu/files/10771916/crafttweaker.log)

**Comment by kaspertfg** [comment]
https://pastebin.com/5BnNzFpL

**Comment by paynejj** [comment]
Here are the logs when starting from scratch

crash report
[crash.txt](https://github.com/tracer4b/nomi-ceu/files/10772375/crash.txt)

latest.log
[latest.log](https://github.com/tracer4b/nomi-ceu/files/10772380/latest.log)

crafttweaker log
[crafttweaker.log](https://github.com/tracer4b/nomi-ceu/files/10772388/crafttweaker.log)

**Comment by IntegerLimit** [comment]
Thanks, i'll have a look at it. Try reporting to Multiblock Tweaker.

**Comment by KuroKenn** [comment]
> Update: The 1.5.1 server files do not work at all. Starting from scratch, no changes made by me, the server crashes with this error:
> 
> Description: Exception in server tick loop
> 
> net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from CraftTweaker2 (crafttweaker) Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/texture/TextureMap

I had the same issue on my other server (linux on the steam deck) solved by manually updating CraftTweaker2 from 4.1.20.683 to 4.1.20.684, test if it's repeatable for you

**Comment by KuroKenn** [comment]
The error when logging in also changed to this
![2023-02-18_14 20 54](https://user-images.githubusercontent.com/20042781/219868242-f6b698ea-55c4-47d9-a04b-a0eea1b8139c.png)

**Comment by paynejj** [comment]
I tried a fresh boot only reinstalling MBT 1.8.2 gtceu. The errors matched KuroKenn's above
I tried a fresh boot only updating crafttweaker2 to .684. The errors matched what I posted earlier. 

I tried a fresh boot installing both. 
1. When I copied the eula from another folder so it did not stop mid boot, I got nomalmode.zs errors.
2. When I let the server stop mid boot to sign the eula, I did not get any chat errors. Here are the logs. 
    - [crafttweaker.log](https://github.com/tracer4b/nomi-ceu/files/10774669/crafttweaker.log)
    - [latest.log](https://github.com/tracer4b/nomi-ceu/files/10774670/latest.log)
3. I then rebooted the server and got errors matching KuroKenn's. Here are the logs
    - [crafttweaker.log](https://github.com/tracer4b/nomi-ceu/files/10774680/crafttweaker.log)
    - [latest.log](https://github.com/tracer4b/nomi-ceu/files/10774681/latest.log)

**Comment by IntegerLimit** [comment]
We're waiting for a MBT patch update to resolve this issue. When that patch is released, we will release 1.5.2.

**Comment by kaspertfg** [comment]
I believe I have found a solution.  In GregTech\src\main\java\gregtech\integration\jei\multiblock\MultiblockInfoRecipeWrapper.java line 487 a static was added in Fix Probable Bugs (https://github.com/GregTechCEu/GregTech/pull/1330).  When this is removed I cant recreate the issue.

**Comment by IntegerLimit** [comment]
Amazing, thanks very much!

**Comment by IntegerLimit** [comment]
Note: I changed the tag to GTCEu problem, as the other tag is for merged prs.

**Comment by IntegerLimit** [comment]
Might be fixed with https://github.com/GregTechCEu/GregTech/pull/1541 or https://github.com/GregTechCEu/GregTech/pull/1544, or maybe both.

**Comment by ALongStringOfNumbers** [comment]
Would anyone having these issues be able to try the file in https://github.com/GregTechCEu/GregTech/pull/1544? Just change the extension of the file from `.zip` to `.jar`

**Comment by dipskii** [comment]
> Would anyone having these issues be able to try the file in [GregTechCEu/GregTech#1544](https://github.com/GregTechCEu/GregTech/pull/1544)? Just change the extension of the file from `.zip` to `.jar`

This fixed the same issue for me.
